Makefile

```diff
@@ -51,6 +53,12 @@ ifeq ($(cc_system_build),yes)
     QEMUBINDIR    := $(BINDIR)
     SYSCONFDIR    := /etc
     LOCALSTATEDIR := /var
+
+    ifeq (,$(installing))
+        # Force a rebuild to ensure version details are correct
+	# (but only for a non-install build phase).
```

Comment:
mix of tabs and spaces on these two comment lines

Reply:
Fixed.